src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -27,11 +28,13 @@
 class TwigDataCollector extends DataCollector implements LateDataCollectorInterface
 {
     private $profile;
+    private $twig;
     private $computed;
 
-    public function __construct(Profile $profile)
+    public function __construct(Profile $profile, Environment $twig)
```

Comment:
That's a BC break. It should be optional. @ro0NL Can you submit a PR to fix this?